run.sh

```diff
@@ -346,6 +349,23 @@ check_plugins() {
   done <<< "$(WP plugin list --field=name)"
 }
 
+check_packages() {
+  if [[ ! "${REQUIRE-}" ]]; then
+    h3 "No package dependencies listed"
+    STATUS SKIP
+    return
+  fi
+
+  h3 "Installing composer packages"
+  touch /app/composer.json
```

Comment:
Also, composer.json could be volumed anywhere: in `/app`, `/app/wp-content`, `/app/wp-content/plugins`, `/app/wp-content/plugins/any-plugin`. So detecting this file to automatically `composer install` it is quite a challenge. Detecting REQUIRE is far easier.